CONTRIBUTING.md

```diff
@@ -57,7 +57,7 @@ section for more details why the development is done directly on fork's `master`
    git push
    ```
 
-1. Your changes should be visible in Katacoda at h&#8203;ttps://katacoda.com/your_katacoda_user
+1. Your changes should be visible in Katacoda at https://katacoda.com/your_katacoda_user
```

Comment:
Adam, this is not a mistake. This is a Zero Width Space character which allows to avoid GitHub to transform this into an hyperlink. https://katacoda.com/your_katacoda_user goes nowhere.